decidim-debates/app/controllers/decidim/debates/debates_controller.rb

```diff
@@ -84,6 +103,10 @@ def report_form
         @report_form ||= form(Decidim::ReportForm).from_params(reason: "spam")
       end
 
+      def close_debate_form
```

Comment:
This doesn't seem to be used

Reply:
It's used here https://github.com/decidim/decidim/blob/feature/close-debates/decidim-debates/app/views/decidim/debates/debates/show.html.erb#L108